pubnub-kotlin/pubnub-kotlin-api/src/jsMain/kotlin/com/pubnub/api/PubNubImpl.kt

```diff
@@ -712,7 +713,26 @@ class PubNubImpl(val jsPubNub: PubNubJs) : PubNub {
         sort: Collection<PNSortKey<PNMembershipKey>>,
         include: MembershipInclude
     ): GetMemberships {
-        TODO("Not yet implemented")
+        return GetMembershipsImpl(
+            jsPubNub,
+            createJsObject {
+                userId?.let { this.uuid = it }
+                this.sort = sort.toJsMap()
+                this.filter = filter
+                this.page = page.toMetadataPage()
+                this.include = createJsObject<PubNubJs.MembershipIncludeOptions> {
+                    this.customFields = include.includeCustom
+                    this.totalCount = include.includeTotalCount
+                    this.channelFields = include.includeChannel
+                    this.customChannelFields = include.includeChannelCustom
+                    this.channelTypeField = include.includeChannelType
+                    this.channelStatusField = include.includeChannelStatus
+                    this.statusField = include.includeStatus
+                    this.typeField = include.includeType
+                }
```

Comment:
perhaps we could extract this into a function like `MembershipInclude.toMembershipIncludeOptions()` if it's used in a few places